<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>675</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SF Pro Text,segoe UI,arial</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font-family: 'SF Pro Text', 'segoe UI', 'arial';
font-weight: 400;
font-size: 14px;
color: #374353;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startDrawing">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 1px solid black;
border-radius:  20px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/resources/icons/play-button.png</normaloff>:/icons/resources/icons/play-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseDrawing">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 1px solid black;
border-radius:  20px;
</string>
    </property>
    <property name="text">
     <string>Tạm dừng</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberOfDrop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectMode">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Hạt mực biến mất khi chạm tường</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hạt mực nảy lại khi chạm tường</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="startX">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="width">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Chiều ngang&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Chiều ngang&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="height">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chiều dọc&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Chiều dọc&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxSpeed">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="radius">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="countInkdropInsideCircle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Kết quả số hạt nằm trong vòng tròn</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Số lượng hạt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Kích thước:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10px;</string>
    </property>
    <property name="text">
     <string>pixel x pixel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Vị trí xuất phát:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10px;</string>
    </property>
    <property name="text">
     <string>pixel x pixel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10px;</string>
    </property>
    <property name="text">
     <string>pixel/seconds</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Tốc độ tối đa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10px;</string>
    </property>
    <property name="text">
     <string>pixel </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Bán kính:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Chế độ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkTimer">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border: none;
border-bottom: 2px solid #25bea0;
border-radius: 0px;
color: #25bea0;
font-size: 16px;
font-weight: 500;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>T/gian k/tra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10px;</string>
    </property>
    <property name="text">
     <string>milliseconds</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="checkResultSet">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'monaco', 'courier new', 'consolas';</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopDrawing">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 1px solid black;
border-radius:  20px;
</string>
    </property>
    <property name="text">
     <string>Dừng</string>
    </property>
   </widget>
   <zorder>label_13</zorder>
   <zorder>label_3</zorder>
   <zorder>startDrawing</zorder>
   <zorder>pauseDrawing</zorder>
   <zorder>numberOfDrop</zorder>
   <zorder>selectMode</zorder>
   <zorder>startX</zorder>
   <zorder>startY</zorder>
   <zorder>width</zorder>
   <zorder>height</zorder>
   <zorder>maxSpeed</zorder>
   <zorder>radius</zorder>
   <zorder>countInkdropInsideCircle</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>checkTimer</zorder>
   <zorder>label_12</zorder>
   <zorder>checkResultSet</zorder>
   <zorder>stopDrawing</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
